astropy/io/fits/convenience.py

```diff
@@ -408,9 +410,11 @@ def writeto(filename, data, header=None, output_verify='exception',
         ``+warn``, or ``+exception" (e.g. ``"fix+warn"``).  See :ref:`verify`
         for more info.
 
-    clobber : bool, optional
-        If `True`, and if filename already exists, it will overwrite
-        the file.  Default is `False`.
+    overwrite : bool, optional
+        If ``True``, overwrite the output file if exists.
```

Comment:
On the comma ;-) 
I read it as "If it is true, then ...", requiring a comma; in particular because there are two present tense verbs in that sentence. ([Additional reasoning](http://english.stackexchange.com/questions/52091/comma-after-introductory-phrases).)